config/routes.rb

```diff
@@ -1,15 +1,15 @@
 Rails.application.routes.draw do
   root 'airlines#index'
 
-  resources :airlines, only:[:index, :new, :create, :show, :post] do
-    resources :reviews
+  resources :airlines, only: [:index, :new, :create, :show, :post] do
+    resources :reviews, only: [:index, :show, :new, :create]
```

Comment:
Im still a little confused here. At no point do you actually use `:index, :show, :new, :create` in a regular reviews controller because it is all done in your API controller. line 5 is pretty much not needed.